modules/installation/proc-installing-rhdh-on-openshift-with-helm-cli.adoc

```diff
@@ -5,10 +5,10 @@ You can use the Helm CLI to install {product} on {ocp-brand-name}.
 
 .Prerequisites
 * You have installed the {openshift-cli} on your workstation.
-* You are logged in to your OpenShift Container Platform account.
-* A user with the OpenShift Container Platform admin role has configured the appropriate roles and permissions within your project to create an application. For more information about OpenShift Container Platform roles, see [Using RBAC to define and apply permissions](https://docs.openshift.com/container-platform/4.15/authentication/using-rbac.html).
-* You have created a project in OpenShift Container Platform. For more information about creating a project in OpenShift Container Platform, see [Red Hat OpenShift Container Platform documentation](https://docs.openshift.com/container-platform/4.15/applications/projects/working-with-projects.html#odc-creating-projects-using-developer-perspective_projects).
-* You have installed the link:https://helm.sh/docs/intro/install[Helm CLI].
+* You are logged in to your {ocp-short} account.
+* A user with the {ocp-short} admin role has configured the appropriate roles and permissions within your project to create an application. For more information about {ocp-short} roles, see link:https://docs.openshift.com/container-platform/4.15/authentication/using-rbac.html[Using RBAC to define and apply permissions].
+* You have created a project in {ocp-short}. For more information about creating a project in {ocp-short}, see link:https://docs.openshift.com/container-platform/4.15/applications/projects/working-with-projects.html#odc-creating-projects-using-developer-perspective_projects[{ocp-brand-name} documentation].
```

Comment:
a specific project?

Reply:
any namespace or project you'd like, though there are recommendations for a standard name like "rhdh" or "developer-hub"

Reply:
We might want to be more specific in the future and say something like _You have created a Red Hat Developer Hub project..._.